sample/src/App.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import {View, Text} from 'react-native';
+import {Provider} from 'react-redux';
+import {NavigationContainer} from '@react-navigation/native';
+import {createStackNavigator} from '@react-navigation/stack';
+
+// Import the Sentry React Native SDK
+import * as Sentry from '@sentry/react-native';
+
+import HomeScreen from './screens/HomeScreen';
+import TrackerScreen from './screens/TrackerScreen';
+import PerformanceTestScreen from './screens/PerformanceTestScreen';
+import EndToEndTestsScreen from './screens/EndToEndTestsScreen';
+
+import {store} from './reduxApp';
+import {version as packageVersion} from '../../package.json';
+import {SENTRY_INTERNAL_DSN} from './dsn';
+
+Sentry.init({
+  // Replace the example DSN below with your own DSN:
+  dsn: SENTRY_INTERNAL_DSN,
+  debug: true,
+  beforeSend: (e) => {
+    if (!e.tags) {
+      e.tags = {};
+    }
```

Comment:
why do we need this, just curiosity

Reply:
Hmm, this was carried over. Seems to have been added in this commit https://github.com/getsentry/sentry-react-native/commit/3bbea7e3b4c1ca17da2739a67c87541a401aebad#diff-a849ca39642da5fbe1b2a91f472275752282a0f5d792bddb67b8fd790df3a1a7R34

Reply:
Removed